docs/application/native/guides/text-input/capi-ui-gesture.md

```diff
@@ -0,0 +1,248 @@
+# Gesture
+
+
+According to the dictionary definition, `Gesture` means a movement or position of the hand, body, head, and face, and it expresses emotion, opinion, and so on.
+
+As there are many gesture types, so there are various methods to recognize gestures on the device.
+For example, hand gestures are recognized, based on gyro sensor data. And, head movements can be detected with a visual camera and video frames.
+According to the targeted gesture, the device and the input sensor, the recognition method can be changed.
+
+In the current Gesture, only wearable device is targeted and only hand gesture is recognized from input sensor data.
+
+The main features of the Gesture API include:
+
+- Getting the supported gesture type
+
+  You can [get the supported gesture type](#info_gesture) to be recognized.
+
+- Setting gesture options
+
+  You can [set options](#set_option) such as a hand gesture type and option.
+
+- Preparing the Gesture service for use
+
+  You can [connect the background Gesture daemon](#prepare) to be able to operate the gesture.
+
+
+<a name="basic_gesture"></a>
+## Basic Gesture Processes
+
+Using Gesture, you can:
+
+- [Create a handle](#create_destroy) and register a error callback function.
+  - Create a Gesture handle which is used for distinguishing your application from other applications also using the Gesture.
```

Comment:
Done

Reply:
> Question. Isn't the "the" needed before the Gesture? To me it seems the Gesture handle is used to distinguish your application from other applications which are using the same Gesture with your application. If my understanding is wrong, please skip my comment.

Thank you for remaining the question. I meant to say that 'from other applications also using Gesture APIs'. And, I'm not sure, but I thought that if 'Gesture' is handled as a proper noun, it will be acceptable to remove 'the'. So, I removed 'the' in front of 'Gesture'. If you have any other opinion, please tell me. Thank you!!